modules/cas_disk/exp_obj.c

```diff
@@ -683,6 +714,7 @@ EXPORT_SYMBOL(casdsk_exp_obj_activated);
 int casdsk_exp_obj_lock(struct casdsk_disk *dsk)
 {
 	struct casdsk_exp_obj *exp_obj;
+	int result;
```

Comment:
How about initializing `result` with `-EBUSY`? This way you could remove the first branch of `if`